website/docusaurus.config.js

```diff
@@ -62,9 +62,9 @@ module.exports = {
           anonymizeIP: true,
         },
         blog: {},
-        theme: {
-          customCss: "/src/css/customTheme.css",
-        },
+        // theme: {
+        //   customCss: "/src/css/customTheme.css",
+        // },
```

Comment:
why are you disabling the CSS theme?

Reply:
Hey. Sorry for not responding. I had problems running the development server while this theme was here. So I removed it just to be able to work. I pushed new changes that does not remove it.